test/cnxml/se/admonition-with-glossary-and-definition.js

```diff
@@ -42,7 +44,7 @@ export const output = `
     </content>
     <glossary>
         <definition>
-            <term>Term</term>
+            <term xmlns:cmlnle="http://katalysteducation.org/cmlnle/1.0" cmlnle:reference="other value">Term</term>
```

Comment:
I'd prefer the namespace to be set on `<document>` (this won't affect tests since CNXML comparison is done on DOM element, not strings), as it is clearer.